src/main/resources/data.sql

```diff
@@ -0,0 +1,55 @@
+INSERT INTO privilege (id, operation, owner, target) VALUES
```

Comment:
I didn't have this sql file run when I started the app. Does it need something else to get it going?